build.sbt

```diff
@@ -376,32 +413,30 @@ lazy val `proxy-core` = (project in file("proxy/core"))
         "io.grpc" % "grpc-netty-shaded" % GrpcJavaVersion,
         // Since we exclude Aeron, we also exclude its transitive Agrona dependency, so we need to manually add it HERE
         "org.agrona" % "agrona" % "0.9.29",
-        "com.typesafe.akka" %% "akka-remote" % AkkaVersion excludeAll (excludeTheseDependencies: _*),
+        akkaDependency("akka-remote"),
+        "com.typesafe.akka" %% "akka-protobuf-v390" % (AkkaVersion + "-proto390") excludeAll (excludeTheseDependencies: _*), // FIXME since akka-protobuf-v3 uses a shaded protobuf-java 3.10.0 which is not supported in native-image yet…
```

Comment:
@patriknw special-cut this for us, as Protobuf-Java 3.10.0 uses VarHandles which are not fully working with GraalVM native-image 20.0.0 (and 20.1.0-dev).

Reply:
@pvlugter @jroper @patriknw We need to decide on how to proceed with this—should we downgrade akka-protobuf to 3.9.0 or should we maintain an `akka-protobuf-v390` artifact, or should we consider having akka depending on (and not shading) protobuf-java 3?

Reply:
It looks like the usage of method handles was removed again in protobuf-java 3.11. So we may be able to instead upgrade akka-protobuf-v3 to 3.11?

Reply:
Also discussed in https://github.com/protocolbuffers/protobuf/issues/6497

Reply:
I can try updating akka-protobuf-v3 to 3.11

Reply:
https://github.com/akka/akka/pull/29014

Reply:
@pvlugter This cannot be fixed until Akka 2.6.6